<launch>
	<arg name="numero_maquinas" default="2"/>




 	<machine name="client" address="rosclient" user="jgarruchof" env-loader="/opt/ros/noetic/env.sh" password="StepInside"/>

	<include file="$(find tfg_javgarfer)/launch/maquinas.launch">
       <arg name="numero_maquinas" value="$(eval arg('numero_maquinas'))"/>
   </include>

	<node machine="client" name="usb_cam" pkg="usb_cam" type="usb_cam_node">
		<param name="pixel_format" value="yuyv"/>
		<param name="image_width" value="1600"/>
		<param name="image_height" value="800"/>
		<param name="framerate" value="3"/>
	</node>

	<node name="cam2stereo" pkg="tfg_javgarfer" type="cam2stereo.py" args="$(arg numero_maquinas)"/> <!-- args="$(arg numero_maquinas)" -->
	


	<include file="$(find tfg_javgarfer)/launch/image_procs.launch">
       <arg name="numero_maquinas" value="$(eval arg('numero_maquinas'))"/>
   </include>



<!--
	<node name="visualizador" pkg="image_view" type="stereo_view">
		<param name="stereo" value="stereo"/>
		<param name="image" value="image_rect"/>
		<remap from="image" to="image_rect"/>
	</node>
-->

</launch>